### PR TITLE
Kannada TN - Created Ordinal Semiotic Class

### PR DESCRIPTION
**Pull request by richa-2002**
# What does this PR do ?

Add a one line overview of what this PR aims to accomplish.

# Before your PR is "Ready for review"
**Pre checks**:
- [x] Have you signed your commits? Use ``git commit -s`` to sign.
- [x] Do all unittests finish successfully before sending PR?
   1) ``pytest`` or (if your machine does not have GPU) ``pytest --cpu`` from the root folder (given you marked your test cases accordingly `@pytest.mark.run_only_on('CPU')`).
   2) Sparrowhawk tests ``bash tools/text_processing_deployment/export_grammars.sh --MODE=test ...``
- [x] If you are adding a new feature: Have you added test cases for both `pytest` and Sparrowhawk [here](tests/nemo_text_processing).
- [x] Have you added ``__init__.py`` for every folder and subfolder, including `data` folder which has .TSV files?
- [ ] Have you followed codeQL results and removed unused variables and imports (report is at the bottom of the PR in github review box) ?
- [x] Have you added the correct license header `Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.` to all newly added Python files?
- [ ] If you copied [nemo_text_processing/text_normalization/en/graph_utils.py](nemo_text_processing/text_normalization/en/graph_utils.py) your header's second line should be `Copyright 2015 and onwards Google, Inc.`. See an example [here](https://github.com/NVIDIA/NeMo-text-processing/blob/main/nemo_text_processing/text_normalization/en/graph_utils.py#L2).
- [ ] Remove import guards (`try import: ... except: ...`) if not already done.
- [ ] If you added a new language or a new feature please update the [NeMo documentation](https://github.com/NVIDIA/NeMo/blob/main/docs/source/nlp/text_normalization/wfst/wfst_text_normalization.rst) (lives in different repo).
- [x] Have you added your language support to [tools/text_processing_deployment/pynini_export.py](tools/text_processing_deployment/pynini_export.py).
  

**PR Type**:
- [x] New Feature
- [ ] Bugfix
- [ ] Documentation
- [ ] Test

If you haven't finished some of the above items you can still open "Draft" PR.

### CODE REVIEW

File: nemo_text_processing/text_normalization/kn/data/ordinals/en_suffixes.tsv [hunk 1]
```diff
@@ -0,0 +1,4 @@
+th	ನೆಯ
+st	ನೆಯ
+nd	ನೆಯ
+rd	ನೆಯ
```
Comment (folivoramanh) [inline nemo_text_processing/text_normalization/kn/data/ordinals/en_suffixes.tsv]: **should-fix · verified**

These four map unconditionally, with nothing tying a suffix to the digit it may follow — so the
grammar accepts English ordinals that don't exist:

```
5st   ->  ಐದನೆಯ
2th   ->  ಎರಡನೆಯ
7nd   ->  ಏಳನೆಯ
4rd   ->  ನಾಲ್ಕನೆಯ
11st  ->  ಹನ್ನೊಂದನೆಯ     (English is 11th, never 11st)
```

In English the suffix is determined by the final digit, with 11/12/13 as the exception: `st` after
…1 (not 11), `nd` after …2 (not 12), `rd` after …3 (not 13), `th` everywhere else. Right now any
digit + any suffix normalizes silently, which means a typo or an unrelated token gets swallowed by
the ordinal class instead of falling through to `word`.

I haven't written a patch because the constraint is fiddly enough that an untested one would be
worse than none — but the repo idiom is to encode it structurally with `pynini.difference` on the
last digit rather than by weight, the way `hi/taggers/roman.py` gates its input set. Worth a test
case for at least `11th` vs `11st` once it's in.

File: nemo_text_processing/text_normalization/kn/data/ordinals/exceptions.tsv [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+6th	ಸಿಕ್ಸ್ತ್
+7th	ಸೆವೆಂತ್
+8th	ಎಯ್ತ್
+9th	ನೈನ್ತ್
+10th	ಟೆನ್ತ್
+11th	ಇಲೆವೆಂತ್
+12th	ಟ್ವೆಲ್ಫ್ತ್
+13th	ಥರ್ಟೀನ್ತ್
+14th	ಫೋರ್ಟೀನ್ತ್
+15th	ಫಿಫ್ಟೀನ್ತ್
```
Comment (folivoramanh) [inline nemo_text_processing/text_normalization/kn/data/ordinals/exceptions.tsv]: **should-fix · verified + linguistic-claim**

This block gives 1st–15th a transliterated **English** reading, but anything from 16 up falls
through to the compositional path and gets a native **Kannada** reading. So the same orthographic
pattern flips reading systems mid-range:

```
14th  ->  ಫೋರ್ಟೀನ್ತ್     (English)
15th  ->  ಫಿಫ್ಟೀನ್ತ್      (English)
16th  ->  ಹದಿನಾರನೆಯ      (Kannada)   <- discontinuity
17th  ->  ಹದಿನೇಳನೆಯ      (Kannada)
```

Same for Kannada digits with an English suffix (`೧೫th` -> ಫಿಫ್ಟೀನ್ತ್, `೧೬th` -> ಹದಿನಾರನೆಯ), and the
split shows up in st/nd/rd too — `1st` -> ಫಸ್ಟ್ but `21st` -> ಇಪ್ಪತ್ತೊಂದನೆಯ.

Your own test file encodes both sides of this: L37-38 assert `೮೮th~ಎಂಬತ್ತೆಂಟನೆಯ` / `88th~ಎಂಬತ್ತೆಂಟನೆಯ`
(Kannada), while this file asserts `15th~ಫಿಫ್ಟೀನ್ತ್` (English). So it isn't a bug I'm inferring —
it's two conventions coexisting.

I don't speak Kannada, so I can't say which is right, and both are defensible: reading Latin-suffixed
ordinals as transliterated English is reasonable for code-switched text, and so is always reading the
number natively. What isn't defensible is the boundary at 15. Could you pick one and make it uniform?

- If **English for Latin suffixes**: the exception list needs to keep going (16th, 20th, 21st, 100th…),
  which argues for generating it rather than enumerating.
- If **Kannada everywhere**: drop the `1st`–`15th` rows here and keep only the genuinely irregular
  `1ನೇ`/`೧ನೇ` -> ಮೊದಲನೆಯ at L1-2.

File: nemo_text_processing/text_normalization/kn/data/ordinals/ending.tsv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+ು
+ಿ
```
Comment (folivoramanh) [inline nemo_text_processing/text_normalization/kn/data/ordinals/ending.tsv:2]: **optional · verified + linguistic-claim**

Nice use of `cdrewrite` to strip the cardinal's final vowel before the suffix — and the two entries
are well chosen: `ು` covers the digits/teens/ties and `ಿ` covers the three `scale_suffixes.tsv` rows
ending that way (ಕೋಟಿ). I checked and every cardinal in `data/numbers/` ends in one of the two.

Except zero: `ಸೊನ್ನೆ` ends in `ೆ`, which isn't here, so nothing gets stripped:

```
0ನೇ  ->  ಸೊನ್ನೆನೆಯ
೦ನೇ  ->  ಸೊನ್ನೆನೆಯ
```

Whether that even matters is your call — "zeroth" is a marginal thing to write, and rejecting `0ನೇ`
outright (so it falls through to `word`) may well be better than normalizing it. But right now it
produces output, and I'd rather flag that it's unreviewed than let it through silently. If ಸೊನ್ನೆನೆಯ
isn't the form a Kannada reader expects, either add `ೆ` here or exclude zero from the ordinal input
set.

File: tests/nemo_text_processing/kn/data_text_normalization/test_cases_ordinal.txt [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+4th~ಫೋರ್ಥ್
+5th~ಫಿಫ್ತ್
+೧೩th~ಥರ್ಟೀನ್ತ್
+೧೪th~ಫೋರ್ಟೀನ್ತ್
+೧೫th~ಫಿಫ್ಟೀನ್ತ್
+೧೦೦೧rd~ಒಂದು ಸಾವಿರದ ಒಂದನೆಯ
+21st~ಇಪ್ಪತ್ತೊಂದನೆಯ
+ಈ ಮಗುವಿನ ಶೈಕ್ಷಣಿಕ ಫಲಿತಾಂಶವು ಯಾವಾಗಲೂ ಇಡೀ ತರಗತಿಯಲ್ಲಿ ೧ನೇ ಸ್ಥಾನದಲ್ಲಿದೆ~ಈ ಮಗುವಿನ ಶೈಕ್ಷಣಿಕ ಫಲಿತಾಂಶವು ಯಾವಾಗಲೂ ಇಡೀ ತರಗತಿಯಲ್ಲಿ ಮೊದಲನೆಯ ಸ್ಥಾನದಲ್ಲಿದೆ
+ಈ ಸಾಲಿನಿಂದ ಕೆಳಕ್ಕೆ ಎಣಿಸಿದಾಗ 5ನೇ ಶಿಯಾವೋ ಮಿಂಗ್.~ಈ ಸಾಲಿನಿಂದ ಕೆಳಕ್ಕೆ ಎಣಿಸಿದಾಗ ಐದನೆಯ ಶಿಯಾವೋ ಮಿಂಗ್.
+ಅಭಿನಂದನೆಗಳು! ನೀವು ನಮ್ಮ ಅಂಗಡಿಯ ೧೦೧ನೇ ಗ್ರಾಹಕರಾಗಿದ್ದೀರಿ.~ಅಭಿನಂದನೆಗಳು! ನೀವು ನಮ್ಮ ಅಂಗಡಿಯ ನೂರ ಒಂದನೆಯ ಗ್ರಾಹಕರಾಗಿದ್ದೀರಿ.
```
Comment (folivoramanh) [inline tests/nemo_text_processing/kn/data_text_normalization/test_cases_ordinal.txt:69]: **should-fix · verified**

81 cases with both scripts, tier boundaries, a 12-digit number and two full sentences — this is
genuinely good coverage, and the whole suite is green (424 passed on a fresh cache). Two gaps, both
of which hide things I've commented on elsewhere:

**1. Nothing covers the 15/16 reading boundary.** `88th` is here but no case sits either side of the
discontinuity, so whichever convention you settle on in `exceptions.tsv`, these pin it down. Every
expected value below is copied from an actual run against this PR:

```suggestion
ಅಭಿನಂದನೆಗಳು! ನೀವು ನಮ್ಮ ಅಂಗಡಿಯ ೧೦೧ನೇ ಗ್ರಾಹಕರಾಗಿದ್ದೀರಿ.~ಅಭಿನಂದನೆಗಳು! ನೀವು ನಮ್ಮ ಅಂಗಡಿಯ ನೂರ ಒಂದನೆಯ ಗ್ರಾಹಕರಾಗಿದ್ದೀರಿ.
16th~ಹದಿನಾರನೆಯ
21st~ಇಪ್ಪತ್ತೊಂದನೆಯ
೧೦೦೦ನೇ~ಒಂದು ಸಾವಿರನೆಯ
```

I've left `15th` out on purpose — what it *should* produce depends on which convention you pick, and
guessing at it would be worse than leaving it to you. (I nearly shipped a guessed thousand-tier form
here too; the real output turned out to be `ಒಂದು ಸಾವಿರನೆಯ`, not what I'd assumed, which is why it's above verbatim from a run.)

**2. Nothing covers malformed English suffixes**, which is why `5st` and `11st` currently normalize
unnoticed. Once `en_suffixes.tsv` is constrained, a guard belongs here — `11th` normalizes today
(`ಇಲೆವೆಂತ್`) while `11st` should ideally be left alone.

**Review by folivoramanh — COMMENTED** [review]
The thing to settle is in `exceptions.tsv`: 1st–15th get a transliterated English reading while 16th and up get a native Kannada one, so `15th` → ಫಿಫ್ಟೀನ್ತ್ but `16th` → ಹದಿನಾರನೆಯ. Your own test file encodes both conventions, so it needs a decision rather than a bug fix. Separately, `en_suffixes.tsv` accepts any digit with any suffix (`5st`, `11st`), which is worth constraining.

File: nemo_text_processing/text_normalization/kn/data/ordinals/en_suffixes.tsv (hunk 1 above)
Comment (richa-2002) [reply nemo_text_processing/text_normalization/kn/data/ordinals/en_suffixes.tsv]: Removed en_suffixes.tsv and the English ordinal suffix handling, as it was allowing incorrect suffix combinations. The ordinal implementation now supports only Kannada ordinal forms.

File: nemo_text_processing/text_normalization/kn/data/ordinals/exceptions.tsv (hunk 2 above)
Comment (richa-2002) [reply nemo_text_processing/text_normalization/kn/data/ordinals/exceptions.tsv]: Made the ordinal handling uniform by selecting the Kannada convention everywhere. Removed the English transliterated ordinal readings and kept Kannada ordinal forms consistently.

File: nemo_text_processing/text_normalization/kn/data/ordinals/ending.tsv (hunk 3 above)
Comment (richa-2002) [reply nemo_text_processing/text_normalization/kn/data/ordinals/ending.tsv:2]: Added 0ನೇ and ೦ನೇ as exceptions in exceptions.tsv with the output ಸೊನ್ನೆಯ, since zero does not follow the existing last-vowel stripping pattern.

File: tests/nemo_text_processing/kn/data_text_normalization/test_cases_ordinal.txt (hunk 4 above)
Comment (richa-2002) [reply tests/nemo_text_processing/kn/data_text_normalization/test_cases_ordinal.txt:69]: Added the missing boundary cases to the test file, including 15ನೇ, 16ನೇ using the Kannada convention consistently.

**Comment by richa-2002** [comment]
Thank you for the review @folivoramanh , I've addressed all the changes u suggested.

Testing

Pytest: All test cases passed — Ordinal.
Sparrowhawk: All test cases passed — testTNOrdinal

File: nemo_text_processing/text_normalization/kn/data/ordinals/exceptions.tsv [hunk 5]
```diff
@@ -0,0 +1,4 @@
+1ನೇ	ಮೊದಲನೆಯ
+೧ನೇ	ಮೊದಲನೆಯ
+೦ನೇ	ಸೊನ್ನೆಯ
+0ನೇ	ಸೊನ್ನೆಯ
```
Comment (folivoramanh) [inline nemo_text_processing/text_normalization/kn/data/ordinals/exceptions.tsv]: **blocker** · verified

The exception table is keyed only on the `ನೇ` spelling, but `kn_suffixes.tsv` accepts `ನೆ` as well — and your own test file uses that variant for regular numbers (`೧೭೮೨ನೆ`, `6789876ನೆ`, `10000000000000ನೆ`), so it is valid input by this PR's own definition. Both exceptions therefore fall through to the general path on the short variant. I ran all four at `c6b8fe5b`:

- `0ನೆ` → `ಸೊನ್ನೆನೆಯ`, `೦ನೆ` → `ಸೊನ್ನೆನೆಯ` — the exact double-suffix artifact this round removed for `ನೇ` (expected `ಸೊನ್ನೆಯ`)
- `1ನೆ` → `ಒಂದನೆಯ`, `೧ನೆ` → `ಒಂದನೆಯ` — not the `ಮೊದಲನೆಯ` your table declares for `೧ನೇ`

Whichever reading is right for `೧ನೆ`, the two spellings shouldn't disagree.

```suggestion
೦ನೇ	ಸೊನ್ನೆಯ
0ನೇ	ಸೊನ್ನೆಯ
1ನೆ	ಮೊದಲನೆಯ
೧ನೆ	ಮೊದಲನೆಯ
೦ನೆ	ಸೊನ್ನೆಯ
0ನೆ	ಸೊನ್ನೆಯ
```

If you'd rather not double every future exception row, the structural alternative is to key `exceptions.tsv` on the digit alone and let `kn_suffixes` consume the suffix generically in `ordinal.py:41-49` — same effect, no combinatorial growth.

File: tests/nemo_text_processing/kn/data_text_normalization/test_cases_ordinal.txt [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+6789876ನೆ~ಅರವತ್ತೇಳು ಲಕ್ಷದ ಎಂಬತ್ತೊಂಬತ್ತು ಸಾವಿರದ ಎಂಟುನೂರ ಎಪ್ಪತ್ತಾರನೆಯ
+10000000000000ನೆ~ಹತ್ತು ಲಕ್ಷ ಕೋಟನೆಯ
+10000001ನೇ~ಒಂದು ಕೋಟಿಯ ಒಂದನೆಯ
+1000ನೇ~ಒಂದು ಸಾವಿರನೆಯ
+೭೦೦೦೦೦ನೇ~ಏಳು ಲಕ್ಷನೆಯ
+ಈ ಮಗುವಿನ ಶೈಕ್ಷಣಿಕ ಫಲಿತಾಂಶವು ಯಾವಾಗಲೂ ಇಡೀ ತರಗತಿಯಲ್ಲಿ ೧ನೇ ಸ್ಥಾನದಲ್ಲಿದೆ.~ಈ ಮಗುವಿನ ಶೈಕ್ಷಣಿಕ ಫಲಿತಾಂಶವು ಯಾವಾಗಲೂ ಇಡೀ ತರಗತಿಯಲ್ಲಿ ಮೊದಲನೆಯ ಸ್ಥಾನದಲ್ಲಿದೆ.
+ಈ ಸಾಲಿನಿಂದ ಕೆಳಕ್ಕೆ ಎಣಿಸಿದಾಗ 5ನೇ ರಾಘವೇಂದ್ರ.~ಈ ಸಾಲಿನಿಂದ ಕೆಳಕ್ಕೆ ಎಣಿಸಿದಾಗ ಐದನೆಯ ರಾಘವೇಂದ್ರ.
+ಅಭಿನಂದನೆಗಳು! ನೀವು ನಮ್ಮ ಅಂಗಡಿಯ ೧೦೧ನೇ ಗ್ರಾಹಕರಾಗಿದ್ದೀರಿ.~ಅಭಿನಂದನೆಗಳು! ನೀವು ನಮ್ಮ ಅಂಗಡಿಯ ನೂರ ಒಂದನೆಯ ಗ್ರಾಹಕರಾಗಿದ್ದೀರಿ.
+ಈ ಪಟ್ಟಿಯ ಆರಂಭದಿಂದ ೧೦೦ನೇ ವ್ಯಕ್ತಿಯವರೆಗೆ ಇರುವ ಎಲ್ಲರೂ ನಿಮ್ಮ ಗುರಿ ಗ್ರಾಹಕರು.~ಈ ಪಟ್ಟಿಯ ಆರಂಭದಿಂದ ನೂರನೆಯ ವ್ಯಕ್ತಿಯವರೆಗೆ ಇರುವ ಎಲ್ಲರೂ ನಿಮ್ಮ ಗುರಿ ಗ್ರಾಹಕರು.
+0ನೇ~ಸೊನ್ನೆಯ
```
Comment (folivoramanh) [inline tests/nemo_text_processing/kn/data_text_normalization/test_cases_ordinal.txt]: **should-fix** · verified

Only `0ನೇ` is covered here. `೦ನೇ` is in `exceptions.tsv` but untested (the repo convention is to test native-digit values in both scripts), and no exception number is tested with the `ನೆ` variant — which is why the bug on `exceptions.tsv` above passes CI. These lines fail today and pass once that fix lands:

```suggestion
0ನೇ~ಸೊನ್ನೆಯ
೦ನೇ~ಸೊನ್ನೆಯ
0ನೆ~ಸೊನ್ನೆಯ
೦ನೆ~ಸೊನ್ನೆಯ
1ನೆ~ಮೊದಲನೆಯ
೧ನೆ~ಮೊದಲನೆಯ
```

**Review by folivoramanh — COMMENTED** [review]
Re-reviewed at `c6b8fe5b`. The previous round's fixes check out: `en_suffixes.tsv` is gone with no dangling references, the English transliterations are out of `exceptions.tsv`, `0ನೇ`/`೦ನೇ` → `ಸೊನ್ನೆಯ` both work, `15ನೇ` was added, and `88th`/`21st` now correctly fall through to `word` — 414/414 pytest pass on a fresh cache and the TN export compiles clean. The one blocker left is that both exceptions are keyed only on the `ನೇ` spelling, so the `ನೆ` variant your own test file uses elsewhere still produces `ಸೊನ್ನೆನೆಯ` for zero and `ಒಂದನೆಯ` (not `ಮೊದಲನೆಯ`) for one. Sparrowhawk was **not** executed here (no Docker) — please paste your `--MODE=test` output.

File: nemo_text_processing/text_normalization/kn/data/ordinals/exceptions.tsv (hunk 5 above)
Comment (richa-2002) [reply nemo_text_processing/text_normalization/kn/data/ordinals/exceptions.tsv]: Done, Fixed.

File: tests/nemo_text_processing/kn/data_text_normalization/test_cases_ordinal.txt (hunk 6 above)
Comment (richa-2002) [reply tests/nemo_text_processing/kn/data_text_normalization/test_cases_ordinal.txt]: Fixed.

**Comment by richa-2002** [comment]
Thank you for the review @folivoramanh , I've addressed all the changes u suggested.

Testing

Pytest: All test cases passed — Ordinal.
Sparrowhawk:

GRAMMARS = tn_grammars
MODE = test
LANGUAGE = kn
INPUT_CASE = lower_cased
CACHE_DIR = /Users/richa/NeMo-text-processing/tools/text_processing_deployment/kn_tn_grammars_lower_cased
OVERWRITE_CACHE = True
FORCE_REBUILD = False
WHITELIST = 
[I] FSTs do not exist, will overwrite cache
[I] Exporting grammars
NeMo-text-processing :: INFO     :: Created /Users/richa/NeMo-text-processing/tools/text_processing_deployment/kn_tn_grammars_lower_cased/classify/tokenize_and_classify.far
INFO:NeMo-text-processing:Created /Users/richa/NeMo-text-processing/tools/text_processing_deployment/kn_tn_grammars_lower_cased/classify/tokenize_and_classify.far
NeMo-text-processing :: INFO     :: Created /Users/richa/NeMo-text-processing/tools/text_processing_deployment/kn_tn_grammars_lower_cased/verbalize/verbalize.far
INFO:NeMo-text-processing:Created /Users/richa/NeMo-text-processing/tools/text_processing_deployment/kn_tn_grammars_lower_cased/verbalize/verbalize.far
[+] Building 1.9s (19/19) FINISHED                        docker:desktop-linux
=> [internal] load build definition from Dockerfile                      0.0s
=> => transferring dockerfile: 2.01kB                                    0.0s
=> [internal] load metadata for docker.io/continuumio/miniconda3:25.3.1  1.8s
=> [internal] load .dockerignore                                         0.0s
=> => transferring context: 2B                                           0.0s
=> [ 1/15] FROM docker.io/continuumio/miniconda3:25.3.1-1@sha256:4a2425  0.0s
=> => resolve docker.io/continuumio/miniconda3:25.3.1-1@sha256:4a2425c3  0.0s
=> CACHED [ 2/15] WORKDIR /workspace                                     0.0s
=> CACHED [ 3/15] RUN echo "deb http://archive.debian.org/debian stretc  0.0s
=> CACHED [ 4/15] RUN apt-get update &&  apt-get upgrade -y &&   apt-ge  0.0s
=> CACHED [ 5/15] RUN conda install conda-build -y                       0.0s
=> CACHED [ 6/15] RUN conda install -c conda-forge thrax=1.3.4 -y        0.0s
=> CACHED [ 7/15] RUN git clone https://github.com/google/re2            0.0s
=> CACHED [ 8/15] RUN cd re2 && git checkout tags/2022-02-01 && make &&  0.0s
=> CACHED [ 9/15] RUN wget https://github.com/protocolbuffers/protobuf/  0.0s
=> CACHED [10/15] RUN tar xzvf protobuf-2.5.0.tar.gz                     0.0s
=> CACHED [11/15] RUN cd protobuf-2.5.0 && ./configure && make && make   0.0s
=> CACHED [12/15] RUN printf  "# Conda lib path \n/opt/conda/lib" > /et  0.0s
=> CACHED [13/15] RUN git clone https://github.com/anand-nv/sparrowhawk  0.0s
=> CACHED [14/15] RUN git clone https://github.com/kward/shunit2.git     0.0s
=> CACHED [15/15] RUN echo "DONE"                                        0.0s
=> exporting to image                                                    0.0s
=> => exporting layers                                                   0.0s
=> => exporting manifest sha256:b227d50820a3fd33099e01858f9d86ff57821cc  0.0s
=> => exporting config sha256:4374fc3d41bec69364989a942a3dce49cb837fe9f  0.0s
=> => exporting attestation manifest sha256:d93ef2e04d4ef3f71df75505052  0.0s
=> => exporting manifest list sha256:efcebf372199be6dc3b2dd8607cb223648  0.0s
=> => naming to docker.io/library/sparrowhawk:latest                     0.0s
-v /Users/richa/NeMo-text-processing/tools/text_processing_deployment/kn_tn_grammars_lower_cased/classify:/workspace/sparrowhawk/documentation/grammars/en_toy/classify -v /Users/richa/NeMo-text-processing/tools/text_processing_deployment/kn_tn_grammars_lower_cased/verbalize:/workspace/sparrowhawk/documentation/grammars/en_toy/verbalize
testTNCardinal
testTNWord
testTNPunctuation
testTNOrdinal
 
Ran 4 tests.
 
OK

**Review by folivoramanh — APPROVED** [review]
